api/rs/build/lib.rs

```diff
@@ -162,6 +162,17 @@ impl CompilerConfiguration {
         };
         Self { config }
     }
+
+    /// Sets the scale factor to be applied to all `px` to `phx` conversions
+    /// as constant value. This is only intended for MCU environments. Use
+    /// in combination with [`Self::embed_resources`] to pre-scale images and glyphs
+    /// accordingly.
+    #[must_use]
+    pub fn with_scale_factor(self, factor: f32) -> Self {
```

Comment:
~~should it perhaps be called `with_const_scale_factor`?~~
(nah, that would send the wrong signal that the application scale factor will be const)